hedera-node/hedera-app/src/test/resources/signedState/MHS/settingsUsed.txt

```diff
@@ -32,7 +32,6 @@ The following are all the settings, as modified by settings.txt, but not reflect
            5000 = timeoutSyncClientConnect
            5000 = timeoutServerAcceptConnect
            1000 = deadlockCheckPeriod
-            500 = sleepHeartbeat
```

Comment:
Is this intentional ?

Reply:
Seems like it is and is approved by platform.

Reply:
It is! I removed unreferenced properties from the record and from the file too.